src/tests/ftest/util/data_mover_test_base.py

```diff
@@ -1122,7 +1082,7 @@ def run_dm_activities_with_ior(self, tool, create_dataset=False, pool=None, cont
                 dst_path=posix_path)
 
             # create cont3
-            cont3 = self.create_cont(pool, oclass=self.ior_cmd.dfs_oclass.value)
+            cont3 = self.add_container(pool, oclass=self.ior_cmd.dfs_oclass.value)
```

Comment:
(pylint-assignment-from-no-return) Assigning result of a function call, where the function has no return
(pylint-unexpected-keyword-arg) Unexpected keyword argument 'oclass' in method call